images/supreme-light-bubble-sign.svg

```diff
@@ -0,0 +1,116 @@
+           sodipodi:nodetypes="cccccc" />
+      </g>
+      <g
+         transform="matrix(-0.99254184,-0.12190444,-0.12190444,0.99254184,240.11406,9.899235)"
+         id="g2898">
+        <path
+           sodipodi:nodetypes="cccccc"
+           inkscape:connector-curvature="0"
+           id="path2890"
+           d="m 51.561721,82.420558 c 4.947956,-11.443184 7.098326,-21.580828 14.843868,-34.329552 9.493782,-5.622726 18.204793,-8.118789 27.427,-9.201498 7.126671,2.373808 12.308391,3.080516 21.780881,7.465028 1.10404,0.693136 0.69653,1.218322 0.75595,1.795389 z"
+           style="fill:#f7cba4;fill-opacity:1;stroke:#000000;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" />
+        <path
+           sodipodi:nodetypes="cccccccc"
+           inkscape:connector-curvature="0"
+           id="path2892"
+           d="m 48.858904,83.743475 c 5.366384,-11.895346 6.688857,-17.883884 19.294825,-34.235058 7.991142,-3.796463 16.427983,-6.850093 26.540203,-7.111425 7.621828,2.178531 12.478218,2.398214 22.931498,6.58236 0.8726,0.840822 1.84364,2.613648 1.84544,3.33007 L 84.9792,60.102611 44.334964,100.0995 c 1.50798,-5.452008 2.598555,-8.190887 4.52394,-16.356025 z"
+           style="fill:#f7cba4;fill-opacity:1;stroke:#000000;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" />
+        <path
+           sodipodi:nodetypes="cccccccccccccccccc"
+           inkscape:connector-curvature="0"
+           id="path2894"
+           d="m 33.67596,131.02562 c 5.48772,-14.53927 3.346546,-22.2973 20.312483,-47.039435 4.217234,-8.750998 8.4141,-13.112423 18.174327,-27.52876 5.177872,-3.059783 7.406931,-5.37245 24.054259,-10.423512 8.359381,0.921136 18.353891,3.244833 22.450641,4.810851 5.46573,4.389796 3.20814,7.613133 -2.13816,10.690782 -5.73623,1.053712 -15.72047,1.766024 -21.916098,-0.801809 -7.297178,6.492517 -11.785906,8.620178 -17.105249,11.75986 0.262799,3.042295 -1.255482,3.660703 -10.690783,24.588795 -1.43813,4.535448 -0.939345,7.488408 1.870887,21.648828 9.433843,3.3496 13.081836,7.66352 18.592007,11.66707 14.081936,-0.94076 21.484876,-0.21176 24.171116,1.69641 3.30321,3.83086 2.92149,7.66172 0.53454,11.49258 -10.76892,2.71498 -16.152225,2.19859 -23.742458,3.00635 -12.564785,-0.79078 -20.091559,-0.26621 -28.107828,-3.00635 l -4.543582,9.62172 c -10.890523,-0.21362 -22.23952,-7.1044 -26.726954,-14.43256 z"
+           style="fill:#f7cba4;fill-opacity:1;stroke:#000000;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" />
+        <path
+           sodipodi:nodetypes="cccccc"
+           inkscape:connector-curvature="0"
+           id="path2896"
+           d="m 51.561722,82.420557 6.636218,4.305141 -14.031651,15.368002 -0.01454,-1.18643 1.614106,-4.912832 z"
+           style="fill:#f7cba4;fill-opacity:1;stroke:none;stroke-width:0.26458332px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" />
+      </g>
+    </g>
+  </g>
+</svg>
```

Comment:
Is the SVG used anywhere?

Reply:
It's not, but I added it as the source of the image, with hopes that some gifted artist would tune it to look better.